README.md

```diff
@@ -2,6 +2,9 @@
 ilib-webos-loctool-qml is a plugin for the loctool allows it to read and localize qml files. This plugins is optimized for webOS platform.
 
 ## Release Notes
+v1.6.0
+* Fixed an issue where didn't handle single quotes properly.
```

Comment:
If it is a bug fix, this should be v1.5.1. If it is a new feature, then it is v1.6.0

Reply:
I updated it